### ISSUE TITLE
Html mirror cursor also cause line-wise operation (e.g. yy dd) duplicated

### ISSUE THREAD

**Issue by Xdminsy** [VSCodeVim/Vim#4423]
**Describe the bug**
With the html mirror cursor feature, we will have a paired cursor on the paired tag when we move our cursor into one tag.

I found other issues #4402 reported this will cause visual mode error. And also if you use yy or Y to copy this line, it will be duplicated. Or when you use dd the line and the line below will both be deleted.

The html mirror cursor feature maybe should be treated differently from the plain multiple (two) cursors, it may cause many errors as two cursors, I think.

Does anybody know other strange behaviors with this feature?

**To Reproduce**
1. Move the cursor to one html tag.
2. Type yy and p or dd

**Expected behavior**
Only one line should be deleted or copied.

**Environment (please complete the following information):**

- Extension (VsCodeVim) version:
- VSCode version: 1.41.1
- OS: Microsoft Windows [Version 10.0.18362.535]

**Comment by J-Fields** [comment]
> The html mirror cursor feature maybe should be treated differently from the plain multiple (two) cursors, it may cause many errors as two cursors, I think.

Yeah, probably. I haven't really looked into this, but I'm not sure *how* we would easily detect that a cursor originated from this feature.
